src/views/Home.jsx

```diff
@@ -1,11 +1,29 @@
+import { Navigate } from 'react-router-dom';
 import './Home.css';
+import { generateToken } from '@the-collab-lab/shopping-list-utils';
+import * as React from 'react';
+
+export function Home({ listToken, setListToken }) {
+	console.log(listToken); // prints 'undefined' even when a token exists in localStorage
+	const [tokenExists, setTokenExists] = React.useState(
```

Comment:
To avoid typing `React.useState`, you can also import `{ useState } from 'react'`.. Kind of a personal preference, but this works as well.